tasks.py

```diff
@@ -43,6 +41,7 @@ def bin_prefix(cmd):
 @task
 def server(host=None, port=5000, debug=True, live=False):
     """Run the app server."""
+    from website import settings
```

Comment:
If I'm understanding, this is a side effect of importing kombu in `defaults.py`. As a result of that, invoke can't be used unless kombu is installed into the environment.

Another option might be to wrap the import and use of kombu in a conditional. That should work, so long as you never have a task that attempts to run the OSF or access the value of `CELERY_QUEUES` before or in the same thread as when kombu is installed.